pandas/core/indexes/numeric.py

```diff
@@ -72,6 +74,19 @@ def __new__(cls, data=None, dtype=None, copy=False, name=None, fastpath=None):
             name = data.name
         return cls._simple_new(subarr, name=name)
 
+    def _validate_dtype(cls, dtype):
+        if not dtype:
```

Comment:
Maybe make this `if dtype is None:` to be extra safe in case another falsey value is passed for `dtype`.